azurerm/internal/services/keyvault/key_vault_key_encrypt_resource.go

```diff
@@ -0,0 +1,94 @@
+package keyvault
+
```

Comment:
I can see there being a value in having a `resource` block for encrypting secrets with a static encrypted blob, and a `data` block for encrypting secrets with a value that changes on every apply, but I only see an `encrypt` resource. 

Could you create an equivalent `encrypt` data block, as if we're modelling it after aws_kms_ciphertext, we should have the ability for users to choose whether to change the encrypted blob frequently or infrequently.

Reply:
is there any scenario that users want to change the encrypted data every time? In the `resource`, we could also change the encrypted data by changing the property `name`